javaparser-symbol-solver-testing/src/test/resources/javaparser_methodcalls_expected_output/com_github_javaparser_CommentsInserter.txt

```diff
@@ -1,49 +1,10 @@
+  Line 36) modifiers.contains(Modifier.PUBLIC) ==> java.util.AbstractCollection.contains(java.lang.Object)
+  Line 38) modifiers.contains(Modifier.PROTECTED) ==> java.util.AbstractCollection.contains(java.lang.Object)
+  Line 40) modifiers.contains(Modifier.PRIVATE) ==> java.util.AbstractCollection.contains(java.lang.Object)
+  Line 46) sortByBeginPosition(nodes, false) ==> com.github.javaparser.utils.PositionUtils.sortByBeginPosition(java.util.List<T>, boolean)
+  Line 53) EnumSet.noneOf(Modifier.class) ==> java.util.EnumSet.noneOf(java.lang.Class<E>)
+  Line 114) cloneNodes(_n.getPackage(), _arg) ==> com.github.javaparser.ast.visitor.CloneVisitor.cloneNodes(T, java.lang.Object)
+  Line 46) setCheck(check) ==> com.github.javaparser.ast.stmt.AssertStmt.setCheck(com.github.javaparser.ast.expr.Expression)
+  Line 49) ((Node) this).tryAddImportToParentCompilationUnit(typeClass) ==> com.github.javaparser.ast.Node.tryAddImportToParentCompilationUnit(java.lang.Class<?>)
+  Line 86) buf.append(indentation) ==> java.lang.StringBuilder.append(java.lang.String)
   Line 52) comments.isEmpty() ==> java.util.TreeSet.isEmpty()
-  Line 63) cu.getChildrenNodes() ==> com.github.javaparser.ast.Node.getChildrenNodes()
```

Comment:
The fact the generated files has less lines does not seem very good

Reply:
Most of them have (a lot) more, and some dont. It might be the wrong version of JP that we're testing on? But I thought I just copied the source, nothing more.

Reply:
Copied the source? What do you mean?

Reply:
When I moved the source from `test/resource` to `test/testsource` to stop it from confusing the IDE.

Reply:
Mmm... we are not trying to load them as resources, right?

Reply:
Nah, I fixed all the path handling. And we don't have a problem with not finding, but a problem with seeing something different. Worth a double check though.